security/guard_authentication.rst

```diff
@@ -49,6 +49,63 @@ Don't forget to generate and execute the migration:
     $ php bin/console make:migration
     $ php bin/console doctrine:migrations:migrate
 
+Next, configure your "user provider" to use this new `apiKey` property:
+
+.. configuration-block::
+
+    .. code-block:: yaml
+
+        # config/packages/security.yaml
+        security:
+            # ...
+
+            providers:
+                your_db_provider:
+                    entity:
+                        class: App\Entity\User
+                        property: apiKey
```

Comment:
the property name is `apiToken`

```suggestion
                        property: apiToken
```

or did i miss something?

Reply:
Yeah, thanks, it was `apiKey` in v3.4 ...